feature/auth/src/commonMain/kotlin/org/mifospay/feature/auth/login/LoginViewModel.kt

```diff
@@ -36,7 +36,7 @@ class LoginViewModel(
         }
     }
 
-    override fun handleAction(action: LoginAction) {
+    public override fun handleAction(action: LoginAction) {
```

Comment:
- don't make it public. In BaseViewModel we have a function `trysendAction` you can use it to send an action.
- sth like this: 
```
viewModel.trySendAction(AccountAction.UserNameChanged(username))
```